cluster/gce/gci/configure-helper-common.sh

```diff
@@ -3309,9 +3271,8 @@ function start-mizar {
     sleep 5
   done
   kubectl label node `hostname` node-role.kubernetes.io/master=""
-  if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
-    start-mizar-scaleout
-  else
+  #For scaleout cluster, mizar will started after all tp and rp ready
```

Comment:
re-ordering the mizar installation seems good; are the checking readinesses of TP & RP able to identify the instance as ready w/o mizar installation initially?

Reply:
yes, tp/rp identified the instance as ready even without chi plugin installed.